UI/Tests/Test.cs

```diff
@@ -1,9 +1,57 @@
 ﻿using NUnit.Framework;
+using UI.Pages;
+using UI.Steps;
 
 namespace UI.Tests
 {
 	[TestFixture]
 	public class Test : BaseTest
 	{
-	}
+        private static MainPage mainPage = new MainPage();
+        private static string username = "AliceTestAcc4587";
```

Comment:
Let's create a new xml(or another extension) file in Core project which will contain this data and name it TestsConfig.xml